### DISCUSSION TITLE
Search

### DISCUSSION THREAD

**Discussion by Bravozealots** [toeverything/AFFiNE#4452] [General]
Search within the text, it seems not. Can you support it, and the search results are not highlighted.

**Comment by houjoe0829** [comment]
Are you referring to the search function within a page?

**Reply by Bravozealots** [reply]
是的，yes。页面搜索功能有吗

**Reply by houjoe0829** [reply]
这个暂时没有哦，不过，在计划里 This is not available at the moment, but it is in the plan

**Comment by houjoe0829** [comment]
Search within the page is not supported yet, but this will be confirmed in future update plans.

**Reply by Yachih2000** [reply]
目前还没有更新页面内的搜索功能吗
